src/ajax-form.js

```diff
@@ -138,13 +138,19 @@ const ajaxForm = async (ev) => {
       const url = new URL(window.location.href);
       let sort;
       let keyword;
+      let startDate;
+      let endDate;
 
       if (body instanceof FormData) {
         sort = body.get('sort');
         keyword = body.get('keyword');
+        startDate = body.get('startDate');
```

Comment:
The form field should be `start_date` not `startDate`, same goes for `endDate` to `end_date`.